src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c

```diff
@@ -4785,7 +4807,7 @@ kds_custat_show(struct device *dev, struct device_attribute *attr, char *buf)
 	sz += sprintf(buf+sz, "ERT scheduler version : 0x%x\n", ert_version(xert));
 	sz += sprintf(buf+sz, "ERT number of submitted commands: %d\n", exec_num_running(exec));
 	sz += sprintf(buf+sz, "ERT scheduler CU state : {");
-	for (idx = 0; idx < exec->num_cus; ++idx) {
+	for (idx = 0; idx < exec->num_cus + exec->num_sk_cus; ++idx) {
```

Comment:
To prevent future developer issues/confusion, please group add parenthesis to this for declaration.  For example:
` for (idx = 0; idx < (exec->num_cus + exec->num_sk_cus); ++idx) {`

Reply:
done